src/ignitus-Shared/Components/InternshipList/index.ts

```diff
@@ -0,0 +1,43 @@
+import styled from '@emotion/styled';
+import * as T from '../emotionStyles/shared'
+import * as C from '../emotionStyles/colors'
+
+export const InternshipTitle = styled(T.Title)`
+  color:orange
+`;
+
+export const SubHeading = styled(T.Title)`
```

Comment:
No need to extend `T.Title` to create `SubTitle` or `secondaryTitle`, you can create a new `const` in `shared.ts`, something like.

export const subTitle = styled.h3`
  // define font-weight & size here accordingly.
`

Reply:
Done